repos/system_upgrade/common/models/satellite.py

```diff
@@ -22,7 +22,7 @@ class SatelliteFacts(Model):
 
     has_foreman = fields.Boolean(default=False)
     """Whether or not foreman is installed on this system"""
-    has_katello_installer = fields.Boolean(default=True)
-    """Whether or not the installer supports Katello additions"""
+    installer_has_systemchecks = fields.Boolean(default=True)
+    """Whether or not the installer supports --disable-system-checks flag"""
```

Comment:
Is it OK to rename the field in the model, when I know we're the only consumer?
We could also add a new field (or keep the old, not-to-accurate, name).

Reply:
no, you should  go with the deprecation process, which means 6month protection that the original field will exists and will have still the same functionality as before. with the point that upstream documentation is updated to mention this one field became deprecated. docstring should be updated as well.

Reply:
I guess I'll just stick with the old name then ;)